.github/workflows/TEST-bdd.yml

```diff
@@ -41,6 +41,7 @@ jobs:
       - run: sudo chmod -R 777 $ARTIFACTS_DIR
       - run: mkdir -p $CUCUMBER_ARTIFACTS_DIR
       - run: sudo chmod -R 777 $CUCUMBER_ARTIFACTS_DIR
+      - run: npm explore dkg-evm-module -- npm run compile;
```

Comment:
There is no need to use explore here, because I added script for this to npm:

`npm run compile-contracts`

Also I think it makes sense to add it as additional step to `prepare` script, so contracts are compile / artifacts are generated on `npm install`, it will be more clear and we will avoid confusion related to why blockchain isn't starting fast